src/components/Swap/Settings/index.tsx

```diff
@@ -33,7 +31,7 @@ export function SettingsDialog() {
           </ThemedText.ButtonSmall>
         </TextButton>
       </Header>
-      <Column gap={1} style={{ paddingTop: '1em' }} ref={setBoundary} padded css={scrollbar}>
+      <Column gap={1} style={{ paddingTop: '1em' }} ref={setBoundary} padded>
```

Comment:
It's not obvious to me that we need this scrolling styling.  I believe we are okay to remove it

Reply:
It used to be necessary (there were more settings), but with the pared down settings menu there should never be a scrollbar 👍